test/verify/check-storage-luks

```diff
@@ -169,192 +170,179 @@ class TestStorage(StorageCase):
         self.dialog({ "type": "luks+ext4",
                       "name": "ENCRYPTED",
                       "passphrase": "vainu-reku-toma-rolle-kaja",
-                      "passphrase2": "vainu-reku-toma-rolle-kaja" })
+                          "passphrase2": "vainu-reku-toma-rolle-kaja" })
```

Comment:
That looks like accidental mis-indentation.

Reply:
Indeed